npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "juju-gui",
-  "version": "1.9.7",
+  "version": "2.0.0",
```

Comment:
`npm shrinkwrap` did that, so I'm assuming it's pulling the version set in `package.json`.